src/main/scala/com.snowplowanalytics.iglu/ctl/commands/Deploy.scala

```diff
@@ -133,13 +133,14 @@ object Deploy {
             bucket     <- cursor.downField("bucketPath").as[String]
             profile    <- cursor.downField("profile").as[String]
             region     <- cursor.downField("region").as[String]
+            skipSchemaLists <- cursor.downField("skipSchemaLists").as[Boolean]
```

Comment:
It's interesting that we've [made the config](https://github.com/snowplow-incubator/igluctl/commit/908eefec1242cb766691ecccea78be9a74b1b7f5#diff-6eb9f2b0ae475da16056a06a06048fcc79adafcaf9465578f0ea79ce6cb9bdb2) a self-describing JSON and even [uploaded](https://github.com/snowplow/iglu-central/issues/838) to Iglu Central, but were never using as such, we don't even check `igluctl_config` in the file. Otherwise this change would require a schema bump.

Do you think it's also worth to change this to HOCON if we're moving away from SDJ for configs? Also the config schema should probably be deleted from resources dir.

Reply:
I'm hesitating....  Normally I'm a big fan of HOCON over SDJ for configs.  However, in this instance it is more like a playbook, rather than a typical configuration file.  It reminds me of [the dataflow runner playbook](https://github.com/snowplow/dataflow-runner/blob/master/config/playbook.json.sample) which is SDJ and unlikely to change.

I certainly don't like the charade of having to make a change in iglu-central just because I added a new configuration field.

I wonder if users will thank us or curse us for making a breaking change to hocon?

I made [this PR](#82) to see what the change looks like.